java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/frames/TruffleStackInfo.java

```diff
@@ -41,19 +41,21 @@
 public final class TruffleStackInfo {
     
     private static final String METHOD_GET_FRAMES_INFO = "getFramesInfo";       // NOI18N
-    private static final String METHOD_GET_FRAMES_INFO_SIG = "([Lcom/oracle/truffle/api/debug/DebugStackFrame;Z)[Ljava/lang/Object;";   // NOI18N
+    private static final String METHOD_GET_FRAMES_INFO_SIG = "([Lcom/oracle/truffle/api/debug/DebugStackFrame;ZZ)[Ljava/lang/Object;";   // NOI18N
     
     private final JPDADebugger debugger;
     private final JPDAThread thread;
     private final ObjectVariable stackTrace;
+    private final boolean supportsJavaFrames;
     private TruffleStackFrame[] stackFrames;
     private boolean includedInternalFrames;
     private boolean areInternalFrames;
 
-    public TruffleStackInfo(JPDADebugger debugger, JPDAThread thread, ObjectVariable stackTrace) {
+    public TruffleStackInfo(JPDADebugger debugger, JPDAThread thread, ObjectVariable stackTrace, boolean supportsJavaFrames) {
```

Comment:
Sigtest reports:
```
 "E5.4 - Removing constructor" : 
constructor public org.netbeans.modules.debugger.jpda.truffle.frames.TruffleStackInfo.init(
  org.netbeans.api.debugger.jpda.JPDADebugger,
  org.netbeans.api.debugger.jpda.JPDAThread,
  org.netbeans.api.debugger.jpda.ObjectVariable
)
```
please add new constructor, keep the old one.

Reply:
Fixed.